iamclient/iamclient.go

```diff
@@ -273,6 +272,62 @@ func (client *Client) Close() (err error) {
  * Private
  **********************************************************************************************************************/
 
+func createPublicConnection(
+	serverURL string, cryptocontext *cryptutils.CryptoContext,
+	insecure bool) (connection *grpc.ClientConn, err error) {
```

Comment:
In this case, Please use next long line formatting
func createPublicConnection(serverURL string, cryptocontext *cryptutils.CryptoContext,insecure bool) (
    connection *grpc.ClientConn, err error) {

Reply:
Done